src/slack.ts

```diff
@@ -972,7 +972,7 @@ ${usernames.join(', ')}`
                             ...message,
                             league: _league,
                             member,
-                            isModerator: _league.isModerator(message.user),
+                            isModerator: _league.isModerator(member.lichess_username),
```

Comment:
I've been meaning to replace all of the string on these objects with something with a better type to get rid of these errors. Nice find. :D